internal/schema/schemas/posit-publishing-schema-v3.json

```diff
@@ -112,63 +112,15 @@
-                  "examples": [5]
-                },
-                "read_timeout": {
-                  "type": "integer",
-                  "minimum": 0,
-                  "maximum": 2592000,
-                  "description": "Maximum number of seconds allowed without data received from a client connection. A value of `0` means a lack of client (browser) interaction never causes the connection to close.",
-                  "examples": [30]
-                },
-                "init_timeout": {
-                  "type": "integer",
-                  "minimum": 0,
-                  "maximum": 2592000,
-                  "description": "The maximum number of seconds allowed for an interactive application to start. Posit Connect must be able to connect to a newly launched application before this threshold has elapsed.",
-                  "examples": [60]
-                },
-                "idle_timeout": {
-                  "type": "integer",
-                  "minimum": 0,
-                  "maximum": 2592000,
-                  "description": "The maximum number of seconds a worker process for an interactive application to remain alive after it goes idle (no active connections).",
-                  "examples": [120]
-                },
-                "max_processes": {
-                  "type": "integer",
-                  "minimum": 1,
-                  "description": "Specifies the total number of concurrent processes allowed for a single interactive application.",
-                  "examples": [5]
-                },
-                "min_processes": {
-                  "type": "integer",
-                  "minimum": 0,
-                  "description": "Specifies the minimum number of concurrent processes allowed for a single interactive application.",
-                  "examples": [1]
-                },
-                "max_conns_per_process": {
-                  "type": "integer",
-                  "minimum": 1,
-                  "description": "Specifies the maximum number of client connections allowed to an individual process. Incoming connections which will exceed this limit are routed to a new process or rejected.",
-                  "examples": [50]
-                },
-                "load_factor": {
-                  "type": "number",
-                  "minimum": 0,
-                  "maximum": 1,
-                  "description": "Controls how aggressively new processes are spawned. The valid range is between 0.0 and 1.0.",
-                  "examples": [0.5]
-                }
+                "connection_timeout": {},
+                "read_timeout": {},
+                "init_timeout": {},
+                "idle_timeout": {},
+                "max_processes": {},
+                "min_processes": {},
+                "max_conns_per_process": {},
+                "load_factor": {}
```

Comment:
Could you elaborate more on these schema changes? Not sure I follow the reason to have these fields empty (and the ones below)?